source/common/http/http1/codec_impl.cc

```diff
@@ -324,6 +347,8 @@ void ConnectionImpl::dispatch(Buffer::Instance& data) {
 
   ENVOY_CONN_LOG(trace, "parsed {} bytes", connection_, total_parsed);
   data.drain(total_parsed);
+
+  maybeDirectDispatch(data);
```

Comment:
Is the idea here that we may have paused during the initial loop and now have to do direct dispatch? Can you add a comment?

Reply:
same question here. Comments would help.